com.unity.render-pipelines.universal/Editor/Deprecated.cs

```diff
@@ -6,3 +10,16 @@ internal class Deprecated
     {
     }
 }
+
+namespace UnityEditor.Rendering.Universal
+{
+    [Obsolete("ForwardRendererDataEditor has been deprecated. Use StandardRendererDataEditor instead (UnityUpgradable) -> StandardRendererDataEditor", true)]
+    [MovedFrom("UnityEditor.Rendering.LWRP")]
+    public class ForwardRendererDataEditor : ScriptableRendererDataEditor
+    {
+    	public override void OnInspectorGUI()
```

Comment:
is this the only public member that existed in this class?

Reply:
Yes.